packages/typescript-estree/tests/lib/withoutProjectParserOptions.test.ts

```diff
@@ -1,20 +1,14 @@
 import { withoutProjectParserOptions } from '../../src';
 
 describe('withoutProjectParserOptions', () => {
-  it('removes all project parser options', () => {
+  it('removes only project parser options', () => {
     const without = withoutProjectParserOptions({
+      comment: true,
       EXPERIMENTAL_useProjectService: true,
       project: true,
     });
-    expect(without.EXPERIMENTAL_useProjectService).toBeUndefined();
-    expect(without.project).toBeUndefined();
-  });
-  it('leaves other parser options intact', () => {
-    const without = withoutProjectParserOptions({
-      EXPERIMENTAL_useProjectService: true,
-      project: true,
+    expect(without).toBe({
```

Comment:
Oh, shoot, sorry, I think I meant `toEqual`.

Reply:
No worries, updated in 8e62e0b22e3d916e1c27d08eb86fe6f5dc2e83ac

(My local test runs were crashing on unrelated errors, I have `NODE_OPTIONS=--max-old-space-size=8192` set in my shell profile, which was causing the nx runner to crash. So I only spotted this in CI 😩 )